package.json

```diff
@@ -61,7 +61,7 @@
   "devDependencies": {
```

Comment:
This PR as-is is currently blocked by:
- #2626
- #2652 

The only other packages being updated are commodity dev dependencies like crossenv, glob, rimraf, and yargs. I'm going to abandon this PR for now.